library/src/main/java/com/vwo/mobile/utils/VWOUtils.java

```diff
@@ -179,7 +179,7 @@ private static String capitalize(String str) {
      * @return {@link Boolean#TRUE} is app is running in debug mode else {@link Boolean#FALSE}
      */
     public static boolean isApplicationDebuggable(@NonNull Context context) {
-        return (0 != (context.getApplicationInfo().flags &= ApplicationInfo.FLAG_DEBUGGABLE));
+        return (0 != (context.getApplicationInfo().flags & ApplicationInfo.FLAG_DEBUGGABLE));
```

Comment:
What error did you encounter while using previews?

Reply:
if we expand the operator &= we get `context.getApplicationInfo().flags = context.getApplicationInfo().flags & ApplicationInfo.FLAG_DEBUGGABLE`.  Its means it method change context.getApplicationInfo().flags. This flag has many settings inside, like FLAG_SUPPORTS_SCREEN_DENSITIES and other screen support flags. And when we use isApplicationDebuggable all flags in getApplicationInfo().flags become disabled. This leads to the fact that the layout is destroyed in some places

Reply:
Got it...thanks for the explanation 👍